.golangci.yml

```diff
@@ -91,7 +91,8 @@ linters-settings:
       - kilometres
 
 linters:
-  disable-all: true
+  disable:
+    - errcheck
   enable:
     - gofmt
```

Comment:
Are any of these linters included by default and can be removed from this list now? (maybe its better to keep these explicitly defined anyway)